starter-core/src/main/java/io/micronaut/starter/feature/ci/workflows/github/templates/javaAction.rocker.raw

```diff
@@ -0,0 +1,52 @@
+@import io.micronaut.starter.application.Project
+@import io.micronaut.starter.options.JdkDistribution
+@import io.micronaut.starter.options.JdkVersion
+@import io.micronaut.starter.options.BuildTool
+
+@args (
+JdkVersion jdkVersion,
+JdkDistribution jdkDistribution,
+BuildTool buildTool
+)
+
+name: Java CI with @buildTool.getTitle()
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+@if (buildTool.equals(BuildTool.MAVEN)) {
+permissions:
+  contents: read
```

Comment:
Out of curiosity: why is this needed?

Reply:
I have a mistake with that (it should be for Gradle, not Maven). Github actions generates that for the `Java with Gradle` action but not the `Java with Maven` action, but I don't why it's included for Gradle. Maybe @melix knows?